charts/proxy/values.yaml

```diff
@@ -0,0 +1,51 @@
+replicaCount: 1
+image:
+  registry: docker.io
+  repository: gomods/athens
+  tag: latest
```

Comment:
Once we push a tag to the repo, we will have a latest (https://hub.docker.com/r/gomods/proxy/tags/). However since the chart is versioned, what do you think about making this a static version? Otherwise it makes rolling back the chart not so helpful.